cfme-performance/tests/workloads/test_capacity_and_utilization.py

```diff
@@ -34,17 +36,18 @@ def test_workload_capacity_and_utilization(request, scenario):
         'scenario': scenario}
     monitor_thread = SmemMemoryMonitor(SSHClient(), scenario_data)
 
-    def cleanup_workload(scenario, from_ts):
+    def cleanup_workload(scenario, from_ts, quantifiers, scenario_data):
         starttime = time.time()
         to_ts = int(starttime * 1000)
         g_url = get_scenario_dashboard_url(scenario, from_ts, to_ts)
         logger.debug('Started cleaning up monitoring thread.')
         monitor_thread.grafana_url = g_url
         monitor_thread.signal = False
         monitor_thread.join()
+        add_workload_quantifiers(quantifiers, scenario_data)
```

Comment:
The quantifiers aren't actually passed to the report generator in any way. Because quantifier data can contain info about what happened in the finalizer (ex. how many VMs are deleted in finalizer, which can happen concurrently with monitor_thread.join() ), it is added after the reports have been generated. The order it all happens is:

the monitor thread is closed, and workload report is generated, while other finalizer events occur
add_workload_quantifiers() is called, and the workload.html file is modified

The add_workload_quantifiers() method requires scenario_data to rebuild the results path and get the appropriate file. If you would like, we can move that method to a different file (putting it in the version_info fixture might be an option)